src/test/java/org/jboss/mjolnir/GitHubClientTest.java

```diff
@@ -0,0 +1,60 @@
+package org.jboss.mjolnir;
+
+
+import org.eclipse.egit.github.core.Team;
+import org.eclipse.egit.github.core.User;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.jboss.mjolnir.githubclient.ExtendedTeamService;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Sample test class illustrating the use of GitHubClient.
+ *
+ * (Not to be run during build process.)
+ *
+ * @author Tomas Hofman (thofman@redhat.com)
+ */
+@Ignore
```

Comment:
Because it's running against real GitHub, I made it only to get some data from GH and to see that the calls are really working. I will remove this one from the repo, it doesn't belong there.